### PR TITLE
Edited the "Add" feature

### PR DESCRIPTION
**Pull request by wnchan**
Edited some classes for the "Add" feature and created new fields.

### CODE REVIEW

**Review by Bearypop — COMMENTED** [review]
Looks good to me!

**Review by neyapraveen — COMMENTED** [review]
LGTM! Test cases need to be changed

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -1,42 +1,40 @@
...
-                new Address("Blk 45 Aljunied Street 85, #11-31"),
-                getTagSet("colleagues"))
+            new Person(new Name("Alex Yeoh"), new Major("Computer Science"), new Year("2"),
+                    new Email("alexyeoh@u.nus.edu"), new Description("Friendly person"),
+                    new SocialMediaLink("https://example.com/alex")),
+            new Person(new Name("Bernice Yu"), new Major("Computer Science"), new Year("2"),
+                    new Email("berniceyu@u.nus.edu"), new Description("Colleague from work"),
+                    new SocialMediaLink("https://example.com/bernice")),
+            new Person(new Name("Charlotte Oliveiro"), new Major("Computer Science"), new Year("2"),
+                    new Email("charlotte@u.nus.edu"), new Description("Neighbor"),
```
Comment (PearlynnT) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:27]: use British spelling, change Neighbor to Neighbour

Comment (wnchan) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java]: I edited this. thanks!

**Pull request merged**